index.js

```diff
@@ -171,4 +179,53 @@ modal.addEventListener('click', (e) => {
   generateProduct();
 });
 
-generateProduct();
\ No newline at end of file
+generateProduct();
+
+fullnameError.style.display = 'none';
+messageError.style.display = 'none';
+emailError.style.display = 'none';
+
+function ResetError(e, b) {
+  setTimeout(() => {
+    e.style.border = 'none';
+    b.style.display = 'none';
+  }, 3000);
+}
+sendBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  const regex = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|.(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-z\-0-9]+\.)+[a-z]{2,}))$/;
+  if (fullname.value === '') {
+    fullname.style.border = '2px solid red';
+    fullnameError.textContent = 'FullName is Required';
+    fullnameError.style.display = 'block';
+    ResetError(fullname, fullnameError);
+  } else if (email.value === '') {
+    email.style.border = '2px solid red';
+    emailError.textContent = 'Email Field Required';
+    emailError.style.display = 'block';
+    ResetError(email, emailError);
+  } else if (!regex.test(email.value)) {
+    email.style.border = '2px solid red';
+    emailError.textContent = 'Email address must be valid and in lower case';
+    emailError.style.display = 'block';
+    ResetError(email, emailError);
+  } else if (message.value === '') {
+    message.style.border = '2px solid red';
+    messageError.textContent = 'Message is Required';
+    messageError.style.display = 'block';
+    ResetError(message, messageError);
+  } else if (message.value.length > 500) {
+    message.style.border = '2px solid red';
+    messageError.textContent = 'Message Body cannot Exceed 500 Characters';
```

Comment:
- [ ] Hello @katfogy, great effort working on the form validation, currently i noticed, the validation is not working as expecting. According to the requirement, user should not be able to submit the form if there is an `uppercase` letter in the `email address` and an error you pop around the button or the input form stating that the user should make his/her email `lowercase.`
I notice i can submit the form with uppercase letter in my email. Kindly fix this before you resubmit for review